system_files/silverblue/usr/share/glib-2.0/schemas/zz0-bluefin-modifications.gschema.override

```diff
@@ -24,13 +24,11 @@ secondary-color='FFFFFF'
 [org.gnome.desktop.interface]
 enable-hot-corners=false
 clock-show-weekday=true
-cursor-theme="Yaru"
-icon-theme="Yaru-blue"
-gtk-theme="Yaru-blue"
 font-antialiasing="rgba"
-font-name="Ubuntu 12"
-document-font-name="Ubuntu 12"
-monospace-font-name="Ubuntu Mono 18"
+font-name="Inter 12"
+document-font-name="Inter 12"
+monospace-font-name="JetBrains Mono 18"
```

Comment:
Should we still use size 18 font by default?

Reply:
I left it at 18 since it was like that before.  I'm not opposed to changing this if we feel it's better suited.

Reply:
Note that I picked that based on my own aging eyeballs, we should pick whatever the best default should be.